sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2025-2026.bpmn

```diff
@@ -2085,7 +2085,7 @@ There is an application limit based on offering weeks and a total program year l
     <bpmn:intermediateThrowEvent id="Event_02iyl2y" name="Total Transportation Costs">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=calculatedDataReturnTransportationCost + calculatedDataAdditionalTransportationAllowance" target="calculatedDataTotalTransportationCost" />
+          <zeebe:output source="=calculatedDataReturnTransportationCost + calculatedDataTotalAdditionalTransportationAllowance" target="calculatedDataTotalTransportationCost" />
```

Comment:
I do not see an AC for this change. Was it discussed during the ticket? - `calculatedDataAdditionalTransportationAllowance` -> `calculatedDataTotalAdditionalTransportationAllowance` in calculation.